Gaijinizer patch/Map217.txt

```diff
@@ -1440,549 +1436,711 @@ I am...!?#GT
+> CONTEXT: Map217/events/5/pages/2/671/Dialogue/kazuya
+"Was"?
 
+D-Don't tell me, she was defeated?#Ron
 > END STRING
 
 > BEGIN STRING
 その逆だよ。
  彼女は悪魔を『倒しすぎた』んだ。
 
  時には魔王クラスのやつだって倒しちまった。
-> CONTEXT: Map217/events/5/pages/2/675/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/675/Dialogue/hyakki
+Quite the opposite, she ended up defeating too
+many demons. She sometimes even defeated Demon
+King-class ones.#Ron
 > END STRING
 
 > BEGIN STRING
 問題だったのは……
 \\C[2]彼女の魔力の根元が、『天使の魔力』だったこと……
 
 \\C[0]そう……アンタと同じさ、\\N[1]。
-> CONTEXT: Map217/events/5/pages/2/680/Dialogue/hyakki < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/680/Dialogue/hyakki
+The problem was that \\C[2]the source of her power was
+angelic magic.\\C[0]
 
+That's right, the same as you, \\N[1].#Ron
 > END STRING
 
 > BEGIN STRING
 どうして、彼女が天使の魔力を得たのかまでは分からない。
 
 本人も、仔猫の段階でネコマタに変化していたらしくて
 自分の素性はよく分かってないらしいからね。
-> CONTEXT: Map217/events/5/pages/2/688/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/688/Dialogue/hyakki
+I don't know how she got the magic power of an
+angel. Even she herself apparently didn't
+know much about her origins as she was supposedly
+transformed into a Nekomata when she was a kitten.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 そして、彼女は……それを知らなかったんだ。
 
  『その時』が来るまで、『平和』を守るために
  その魔力を使い続けた………
-> CONTEXT: Map217/events/5/pages/2/693/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/693/Dialogue/hyakki
+And so, not knowing what was going to eventually
+happen to her, she continued to protect peace
+until "that" time came.#RonLocalization
 > END STRING
 
 > BEGIN STRING
 その『結果』が、どういうことになるか……
 
 アンタはよく知っているはずだよ。
-> CONTEXT: Map217/events/5/pages/2/698/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/698/Dialogue/hyakki
+I think you know well what the end result was.#Ron
 > END STRING
 
 > BEGIN STRING
 で、でも……！
  彼女はあんなに元気そうだったじゃないか！！
 
 とても『暴走』しているようには……！！
-> CONTEXT: Map217/events/5/pages/2/702/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/702/Dialogue/kazuya
+B-But, wasn't she really happy back then?!
 
+How could she end up going on a rampage?!#RonIffy
 > END STRING
 
 > BEGIN STRING
 ……………レイナと同じだよ。
 
 彼女は間一髪のところで、仲間たちの力によって
 魔力の渦から引き上げられた
-> CONTEXT: Map217/events/5/pages/2/708/Dialogue/hyakki < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/708/Dialogue/hyakki
+It was the same as with Reina.
 
+She was pulled out from a magic vortex by her
+friends in the nick of time.#Ron
 > END STRING
 
 > BEGIN STRING
 だけど………その引き換えに多くの犠牲が出た……
 
 直前まで安定していて、魔力の暴走も小規模だった
  アンタほどうまくはいかなかったんだよ
-> CONTEXT: Map217/events/5/pages/2/713/Dialogue/hyakki < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/713/Dialogue/hyakki
+But there were a lot of victims left behind in
+exchange for that.
 
+
+Though she was stable right until the last
+minute, and the scale of her rampage was small,
+things didn't work out as well as they did for
+you.#RonIffy
 > END STRING
```

Comment:
Minor rewording for flow (first part):
```
But in exchange...

A lot of sacrifices were made.
```

DeepL has an interesting interpretation of the second part, which might fit better: 
![image](https://user-images.githubusercontent.com/48231102/125568747-15981bc8-f35d-4c55-a1f2-062322627144.png)